pkg/ipamfetcher/customer.go

```diff
@@ -64,11 +72,50 @@ func (d *Device42CustomerFetcher) FetchCustomers(ctx context.Context) ([]domain.
 	}
 	customers := make([]domain.Customer, 0, len(getCustomersResponse.Customers))
 	for _, customer := range getCustomersResponse.Customers {
+		businessUnit := customer.CustomFields.GetValue("Description")
+		if businessUnit == "" {
+			// fallback to customer name
+			businessUnit = customer.Name
+		}
 		customers = append(customers, domain.Customer{
 			ID:            strconv.Itoa(customer.ID),
-			ResourceOwner: customer.ContactInfo,
-			BusinessUnit:  customer.CustomFields.GetValue("Description"),
+			ResourceOwner: getResourceOwner(customer),
+			BusinessUnit:  businessUnit,
 		})
 	}
 	return customers, nil
 }
+
+func getResourceOwner(customer customer) string {
+
+	/*
```

Comment:
Question: Is this order still applicable to people other than use? If not, can we mention that? Or does it not matter

Reply:
I thought about that.  Honestly, this PR leaks Atlassian's internal implementation usage of IPAM into this open source project.  To do it right, I'd need to make the priority ordering and names of "type"s configurable via env var.  I can do that.